contents/blog/brand.md

```diff
@@ -0,0 +1,128 @@
+---
+date: 2023-06-25
+title: How to brand your startup
```

Comment:
maybe more like...

How to actually brand your startup
How to brand your startup so it isn't boring

just to give it a bit more oomph